### DISCUSSION TITLE
Creating a Readme.md template for packages

### DISCUSSION THREAD

**Discussion by younesStrittmatter** [AutoResearch/autora#374] [Core Contributions]
When using the autora-theorist-template, we should add a hint to change the README to a description of the package. Also should we have a README.md template for the packages developed by our team to have consistency? Should we have a README-TEMPLATE in the autora-theorist-template anyway?

**Comment by musslick** [comment]
I think having a readme template would be very useful. And then, we should add a note in the documentation (it should already be in the general readme of the theorist-template repo) to change the internal readme.
